pipeline.config

```diff
@@ -0,0 +1,53 @@
+#
+# (C) Copyright IBM Corp. 2021, 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+INSIGHTS:
+  ID: "9b253889-dee0-47f4-9ea0-402306239aaa"
+CI:
+  DOCKER_IMAGE_NAME: "cohort-cql-translation-image"
+  UMBRELLA_REPO_PATH: "https://github.ibm.com/watson-health-cohorting/wh-cohorting-umbrella.git"
+  REGISTRY_NAMESPACE: "cdt-provider-cohort-rns"
+  POLICY_NAME: "CI"
+  NOLATEST: "true"
+  REPORT_PATH: "lcov.info"
+  BUILDCACHE: "false"
+  REPLACEARGS: "true"
+  VAULT_KEYS_TO_INJECT: "IBM_GITHUB_PUBLIC_USERNAME,IBM_GITHUB_PUBLIC_TOKEN"
```

Comment:
I don't think it will hurt anything to have these here, but these vault keys are probably not used. They are values stored in Key Protect in the cloud which the toolchain delivers as env. var. when the toolchain runs. These 2 were used to pass the github username and access token to the maven build in .toolchain.maven.settings.xml so that we could download from github packages.